### DISCUSSION TITLE
My Old CP/M Floppies

### DISCUSSION THREAD

**Discussion by RWoerz** [keirf/flashfloppy#895] [General]
I need some help. I have a old CP/M machine and the floppies that went with them. I need to make sure FlashFloppy can emulate the format. MY drives are DS/DD 96 tpi 5.25". There is a section of the BIOS that defined the how the disk was laid out called the "diskdef". My diskdef looks like this. Can someone tell me if this will work. I looked a ff.cfg and got lost in a hurry.

diskdef GNAT_DSDD
  seclen 512          Sector length
  tracks 160           Total Number of tracks
  sectrk 10             Sectors per track
  blocksize 2048    Block size
  maxdir 128          Max Director Size
  boottrk 2             Start of boot track
  os 2.2                   CP/M version number 2.2
end

**Comment by keirf** [comment]
Yes, definitely. You would define the required format via IMG.CFG file.

Even better would be to dump the original disks you have using a device such as Greaseweazle. This would mean:
(a) We could see definitively the on-disk layout and emulate it with correct parameters in FlashFloppy's IMG.CFG
(b) You could generate IMG files from your original disks and use those files with FlashFloppy
(c) Alternatively you could generate HFE files from your original disks and use those with FlashFloppy with no need for IMG.CFG

**Reply by RWoerz** [reply]
Thanks I'll have a look at IMG.CFG. I have a KryoFlux and did a dump and sent it to them and that's where I got my data.
